src/code_80004740.c

```diff
@@ -116,8 +116,8 @@ void func_80004C30(u32 *arg0, animation_type_2 *arg1, s16 arg2) {
     D_80162D7A = 0;
     D_80162D76 = 0;
     for (someIndex = 0; someIndex < 3; someIndex++) {
-        if (arg2 < (*temp_v0)[someIndex].some_limiter) {
-            some_offset = arg2;
+        if (time_cycle < (*temp_v0)[someIndex].some_limiter) {
```

Comment:
Consider `timer` if it's just a timer. That like goes up and then resets to zero.

Reply:
I don't document animation so much for now but I will see when I document more chain chomps and penguin to find all correct name

Reply:
kk